qiskit/ignis/verification/tomography/fitters/gateset_fitter.py

```diff
@@ -0,0 +1,662 @@
+# -*- coding: utf-8 -*-
+#
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=missing-docstring
+
+"""
+Quantum gate set tomography fitter
+"""
+
+import itertools
+from typing import Union, List, Dict, Tuple, Optional
+import numpy as np
+import scipy.optimize as opt
+from qiskit.result import Result
+from qiskit.quantum_info import Choi, PTM
+from ..basis.gatesetbasis import StandardGatesetBasis, GateSetBasis
+from .base_fitter import TomographyFitter
+
+
+class GatesetTomographyFitter:
```

Comment:
This class needs a doc string for API docs that explains how it is used, and we could add a short example based on your Jupyter notebook example.

Reply:
Done in e642077